pom.xml

```diff
@@ -562,4 +563,4 @@
       </plugin>
     </plugins>
   </build>
-</project>
\ No newline at end of file
+</project>
```

Comment:
This change seems unnecessary. You should remove the space. Also, it is probably best to put your module at the end of the module list.

Reply:
There is not space. Odd.